### PR TITLE
Install instructions and demo data

### PR DESCRIPTION
**Pull request by anupriyatripathi**
The README file is definitely WIP but perhaps good for a start

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,99 @@
...
+```
+
+Download a small feature table and MGF file using:
+
+```bash
+wget https://github.com/biocore/q2-chemistree/demo/feature-table.biom
+wget https://github.com/biocore/q2-chemistree/demo/sirius.mgf
+```
+
+Once you [import](https://docs.qiime2.org/2018.8/tutorials/importing/) these files into the appropriate QIIME2 artifact format, you can follow these steps to generate your tree. Alternatively, you can proceed by directly downloading the corresponding artifact files as follows:
```
Comment (ElDeveloper) [inline README.md]: I suggest that we only include the BIOM and the MGF files, and that we remove the qza files from the repository. Also include commands to import the data:

```bash
qiime tools import --type MassSpectrometryFeatures --....
qiime tools import --type FeatureTable[Frequency] ...
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,99 @@
...
+
+```bash
+mkdir demo-chemistree
+cd demo-chemistree
+```
+
+Download a small feature table and MGF file using:
+
+```bash
+wget https://github.com/biocore/q2-chemistree/demo/feature-table.biom
```
Comment (ElDeveloper) [inline README.md]: This command is going to fail as GitHub doesn't allow for direct downloads. To do that you would need the *raw* URL. For example if you want to wget chemistree's readme you would do something like this:

```bash
wget https://raw.githubusercontent.com/biocore/q2-chemistree/master/README.md
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,99 @@
...
+Once QIIME2 is [installed](https://docs.qiime2.org/2018.2/install/), activate your QIIME2 environment and install q2-chemistree following the steps below:
+
+```bash
+git clone https://github.com/biocore/q2-chemistree.git
+cd q2-chemistree
+pip install .
+qiime dev refresh-cache
+```
+
+We also need to download SIRIUS, a software-framework for de-novo identification of metabolites. q2-chemistree uses molecular substrucures predicted by this framework to build a hierarchy of the MS1 features in a dataset. SIRIUS is freely available [here](https://bio.informatik.uni-jena.de/software/sirius/).
```
Comment (ElDeveloper) [inline README.md]: In the first mention of SIRIUS can you make that word a hyperlink to their homepage?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,99 @@
...
+
+```bash
+qiime chemistree match-table --i-tree demo-chemisTree.qza \
+  --i-feature-table feature-table.qza \
+  --o-filtered-feature-table filtered-feature-table.qza
+```
+
+This filters the MS1 table to include only the MS1 features with molecular fingerprints. The resulting table is also of type `FeatureTable[Frequency]`.
+
+Thus, using these steps, we can generate a tree relating MS1 features in mass-spectrometry dataset along with a matched feature table. These can be used as inputs to perform [UniFrac-based](10.1038/ismej.2010.133) [alpha-diversity](https://docs.qiime2.org/2018.8/plugins/available/diversity/alpha-phylogenetic/) and [beta-diversity](https://docs.qiime2.org/2018.8/plugins/available/diversity/beta-phylogenetic/) analyses.
```
Comment (ElDeveloper) [inline README.md]: The paper that you want to link is this one:
https://aem.asm.org/content/71/12/8228

**Comment by anupriyatripathi** [comment]
addressed!

**Comment by ElDeveloper** [comment]
Looks great, thanks @anupriyatripathi! :shipit:

**Pull request merged**